mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoInsertMethodGenerator.java

```diff
@@ -125,6 +125,13 @@ public DaoInsertMethodGenerator(
         statementName);
 
     List<? extends VariableElement> parameters = methodElement.getParameters();
+    VariableElement runtTimeAttributeParam = findRuntimeAttributesParam(methodElement);
+    if (runtTimeAttributeParam != null) {
+      parameters = parameters.subList(0, methodElement.getParameters().size() - 1);
+      insertBuilder.addStatement(
+          "boundStatementBuilder = DaoBase.populateBoundStatementWithAttributes(boundStatementBuilder,$L)",
```

Comment:
```suggestion
          "boundStatementBuilder = populateBoundStatementWithAttributes(boundStatementBuilder,$L)",
```
It's not necessary to prefix the method since it's on the parent class.

Unless you think it helps readability, but then we should inject `DaoBase` with a `$T`, and also do it everywhere we use similar methods.